sdk/rust/nym-sdk/src/mixnet/config.rs

```diff
@@ -0,0 +1,36 @@
+use client_core::config::DebugConfig;
+use network_defaults::mainnet;
+use url::Url;
+
+pub struct Config {
```

Comment:
I wonder if with this config we would be somehow able to refactor the "main" `Config`s for the clients and get rid of that `PhantomData`